pegasus/sites.v3/code.org/views/congrats_continue.haml

```diff
@@ -1,17 +1,26 @@
-#continue{:style=>"background-color:grey; color:white; overflow:hidden; padding:20px; text-align:center"}
-  
-  %p{:style=>"font-size: 18px; font-family: 'Gotham 5r', sans-serif;"}
-    %b=I18n.t :continue_learning_header
-
-  %a{:href=>'http://studio.code.org'}
-    %button.button-color-share{:style=>'margin:10px'}
-      %b=I18n.t :learn_next_level
-
-  %a{:href=>'/learn/beyond'}
-    %button.button-color-share{:style=>'margin:10px'}
-      %b=I18n.t :learn_online
+%link{href: "/css/congrats-continue.css", rel: "stylesheet"}
 
+%h2=I18n.t :continue_learning_header
+%div.image-collection
+  %a.white-link{:href=>'http://studio.code.org'}
+    %div.choice
+      %div.choice-container
+        %div.choice-label.purple=I18n.t :learn_code_studio
```

Comment:
Can we lowercase "course" since it looks like sentence case w/ brands capitalized?